__tests__/integration/query.test.ts

```diff
@@ -462,11 +487,12 @@ describe("query can encode / decode QueryValue correctly", () => {
           }
         })`);
     } catch (e) {
-      if (e instanceof TypeError) {
-        expect(e.name).toBe("TypeError");
+      if (e instanceof ClientError) {
+        expect(e.name).toBe("ClientError");
         expect(e.message).toBe(
-          "Passing undefined as a QueryValue is not supported"
+          "A client level error occurred. Fauna was not called."
         );
+        expect(e.cause).toBeDefined();
```

Comment:
a `ClientError` implies its our fault - a bug in the driver.

This is a regression if we are now treating bad input as a driver bug.

Reply:
I reverted this back.